auth-web/src/stores/business.ts

```diff
@@ -343,6 +343,26 @@ export const useBusinessStore = defineStore('business', () => {
         addBusinessTypeforSP(filingBody, business)
```

Comment:
Since I'm looking at this code, I think lines 331-333 should be in the filing object (line 325+) not in the registration object. I guess the Legal API just ignores the business object currently.

Reply:
Never mind. That's for the NOB.

Although, the registration filing object probably should have a business object...

Reply:
Kevin, can you fix this along the way, please?